cmd/goal/ledger.go

```diff
@@ -52,3 +60,29 @@ var supplyCmd = &cobra.Command{
 		fmt.Printf("Round: %v\nTotal Money: %v microAlgos\nOnline Money: %v microAlgos\n", response.Round, response.TotalMoney, response.OnlineMoney)
 	},
 }
+
+var rawBlockCmd = &cobra.Command{
+	Use:   "rawblock",
```

Comment:
nitpick: should say "rawblock [round number]"

Reply:
thanks, fixed